### PR TITLE
Remove usage of `wslu`

### LINKED ISSUE #198: Error launching chrome from WSL
I'm getting an error when trying to use open@7.3.0 in WSL in a project built with @open-wc, @web/dev-server

**This error doesn't happen with open@7.2.1.**

**Steps to reproduce:**

1. Create a new project using:

`npm init @open-wc --type scaffold --scaffoldType wc --features linting testing demoing --typescript true --scaffoldFilesFor testing demoing --tagName ui-element --writeToDisk true --installDependencies npm
cd ui-element
`
2. run with npm start

The following error is displayed.  

[web-dev-server] Error: spawn wslvar ENOENT
[web-dev-server]     at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
[web-dev-server]     at onErrorNT (internal/child_process.js:469:16)
[web-dev-server]     at processTicksAndRejections (internal/process/task_queues.js:84:21) {   
[web-dev-server]   errno: 'ENOENT',
[web-dev-server]   code: 'ENOENT',
[web-dev-server]   syscall: 'spawn wslvar',
[web-dev-server]   path: 'wslvar',
[web-dev-server]   spawnargs: [ 'systemroot' ],
[web-dev-server]   cmd: 'wslvar systemroot',
[web-dev-server]   stdout: '',
[web-dev-server]   stderr: ''
[web-dev-server] }

### PR DESCRIPTION
**Pull request by anaisbetts**
This PR is simplified version of #198 - Windows 10 does not allow installing the OS to locations other than `C:\Windows` so fetching `%systemroot%` is unnecessary here (and since WSL is exclusive to Windows 10, we do not need to worry about Win7 here). WSL also handles marshaling paths between WSL and Windows, so we don't need to convert this path manually.

Fixes #198
Fixes #200
Fixes #204

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,8 +49,10 @@ module.exports = async (target, options) => {
 			cliArguments.push('-a', app);
 		}
 	} else if (process.platform === 'win32' || (isWsl && !isDocker())) {
-		const windowsRoot = isWsl ? await wslGetWindowsEnvVar('systemroot') : process.env.SYSTEMROOT;
-		command = String.raw`${windowsRoot}\System32\WindowsPowerShell\v1.0\powershell${isWsl ? '.exe' : ''}`;
+		command = isWsl ?
+			'/mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe' :
+			`${process.env.SYSTEMROOT}\\System32\\WindowsPowerShell\\v1.0\\powershell`;
```
Comment (anaisbetts) [inline index.js:54]: This line is for standard Win32, where we may be running under Win7 where installing to a different location may be possible - luckily we can just fetch %SYSTEMROOT% here

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by jonaskuske** [comment]
Seeing this too late, but:

> Windows 10 does not allow installing the OS to locations other than C:\Windows so fetching `%systemroot%` is unnecessary here

The Windows location **_is configurable_** (in WSL) – so while this PR hard-coded `/mnt/c/`, it could also be just `/c/` or `/windows/c/` or anything else! The `wslpath` utility from `wslu` (that was removed by this PR) handled this case, so did the previous PR (that was closed in favor of this PR). We'll need an alternative – I guess parsing `/etc/wslconf` in JS as suggested [in the review here](https://github.com/sindresorhus/open/pull/204/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R27) is still an option?

**Comment by anaisbetts** [comment]
Why would fetching the Windows version of systemroot help with that though?

**Comment by jonaskuske** [comment]
@anaisbetts 
When WSL was first introduced, mounting your drives directly into `/` was necessary if you wanted to run Docker, that's why it's still set up this way on some of my devices. Don't know if any (non-legacy) reasons to do so remain... But it's definitely still supported, even now with WSL2: https://docs.microsoft.com/en-US/windows/wsl/wsl-config#automount

I'll just submit a PR to read `root` if specified when I find some time :)

**Pull request merged**